packages/linux_sources.rb

```diff
@@ -3,12 +3,17 @@
 class Linux_sources < Package
   description 'Sources for the Linux kernel'
   homepage 'https://kernel.org/'
-  version '3.18'
-  source_url 'https://www.kernel.org/pub/linux/kernel/v3.x/linux-3.18.tar.xz'
-  source_sha256 'becc413cc9e6d7f5cc52a3ce66d65c3725bc1d1cc1001f4ce6c32b69eb188cbd'
+  version '4.14'
+  source_url 'https://www.kernel.org/pub/linux/kernel/v4.x/linux-4.14.tar.xz'
+  source_sha256 'f81d59477e90a130857ce18dc02f4fbe5725854911db1e7ba770c7cd350f96a7'
 
   def self.install
     linux_src_dir = CREW_DEST_PREFIX + '/src/linux'
+    system 'wget', 'https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay/+archive/refs/heads/release-R75-12105.B/sys-kernel/linux-headers/files.tar.gz'
+    system 'tar', 'xpf', 'files.tar.gz'
+    system "sed -i -e 's,a/,,g' -e 's,b/,,g' *.patch"
```

Comment:
This is needed to allow the patches to be applied with `-Np0 -i` because some paths are prefixed with `a/` or `b/`.

Reply:
Have you tried `-Np1`?

Reply:
@uberhacker Not all the patches have the path prefix.